src/Raven.Server/Documents/Queries/QueryMetadata.cs

```diff
@@ -297,6 +299,16 @@ private static void ThrowUnhandledExpressionTypeInSelect(OperatorType expression
             throw new InvalidQueryException($"Unhandled expression of type {expressionType} in SELECT clause", queryText, parameters);
         }
 
+        private static void ThrowMissingSelectClauseInGroupByQuery(string queryText, BlittableJsonReaderObject parameters)
+        {
+            throw new InvalidQueryException("Query having GROUP BY needs to have at least one aggregation operation defined in SELECT such as count() or sum()", queryText, parameters);
```

Comment:
Why is that? 
`select Country from Users group by Country` is valid way to get all the countries.